react-frontend/src/components/Resources/resources.js

```diff
@@ -4,19 +4,23 @@ import ForDesigners from './fordesigners';
 import ForDevelopers from './fordevelopers';
 import DocumentTitle from 'react-document-title';
 import Guides from './guides';
-import SimpleBanner from '../PageElements/BannerSimple/bannerSimple';
+import BannerSideImage from '../PageElements/BannerSideImage/bannerSideImage';
```

Comment:
The BannerSideImage folder seems to contain only one file. Do we expect more files in this folder in future?

Reply:
Good idea, the different banners should probably all be in one folder.